app/assets/javascripts/admin/addon/components/schema-theme-setting/editor.gjs

```diff
@@ -242,24 +243,38 @@ export default class SchemaThemeSettingEditor extends Component {
     }
   }
 
-  fieldDescription(fieldName) {
+  descriptions(fieldName, key) {
+    // The `property_descriptions` metadata is an object with keys in the following format as an example:
+    //
+    // {
+    //   some_property.description: <some description>,
+    //   some_property.label: <some label>,
+    //   some_objects_property.some_other_property.description: <some description>,
+    //   some_objects_property.some_other_property.label: <some label>,
+    // }
```

Comment:
❤️ thanks, this helps. I just noticed `spec/fixtures/theme_locales/objects_settings/en.yaml` too which is a nice 'live document'